index.bs

```diff
@@ -967,6 +970,39 @@ A string or dictionary identifying a cryptographic algorithm and optionally a se
 defined in [[!WebCryptoAPI]].
 
 
+## Algorithm to Relax the Same-Origin Restriction ## {#algo-relax-same-origin-restriction}
+
+In general, the same-origin policy prevents web applications from interacting
+within user agents, unless they have the same origin. However, there are many
+specific facets and exceptions to this overall policy.
```

Comment:
personally, I think this spec will have readers that are not so clued-in to web security and the same origin policy, thus I'm inclined to provide a modicum of context (tho the above could certainly be polished).